hack/make-files.mk

```diff
@@ -18,6 +18,10 @@ data:
     pkg: $(CSIADDONS_SUBSCRIPTION_PACKAGE)
     scaleUpOnInstanceOf:
       - cephclusters.ceph.rook.io
+  SNAPSHOT_CONTROLLER: |
+    channel: $(ODF_SNAPSHOT_CONTROLLER_SUBSCRIPTION_CHANNEL)
+    csv: $(ODF_SNAPSHOT_CONTROLLER_SUBSCRIPTION_CSVNAME)
+    pkg: $(ODF_SNAPSHOT_CONTROLLER_SUBSCRIPTION_PACKAGE)
```

Comment:
All other things looks good to me, Now the only thing left is when should we scale this up, on presence of cephcluster or storagecluster or noobaa?